content/blog/2020-11-18-análise-da-covid-19-em-3-cidades-do-estado-do-paraná.Rmd

```diff
@@ -0,0 +1,120 @@
+                                     "Confirmados p/ 100k habitantes: ", confirmed_per_100k_inhabitants))) +
+  xlab("Dias/Meses") +
+  ylab("Confirmados") +
+  geom_point(aes(y = confirmed_per_100k_inhabitants), size = 0.4)
+ggplotly(graph.confirmados.p.100k, tooltip = "text")
+```
+
+  Esse controle por 100 mil habitantes tem como objetivo permitir a comparação entre locais com diferentes tamanhos de população e neutralizando o crescimento populacional, nos concedendo a comparação da proliferação do vírus a médio e longo prazo. Assim sendo, notamos a diferença entre as mesmas cidades da figura anterior, na qual Cascavel recebe sua notoriedade chegando próximo aos 2620 casos (por 100k/habitantes).
+
+
+# Óbitos para casos confirmados
+
+```{r echo=FALSE,message=FALSE,warning=FALSE}
+#ÓBITOS GERAL
+
+graph.obitos <- df.cidades.analisadas %>% 
+  ggplot(mapping = aes(fill = city,x = date, y = deaths, color = city,
+                       text = paste0(city, "\n",
+                                     date, "\n", 
+                                     "Óbitos: ", deaths))) +
+  xlab("Dias/Meses") +
+  ylab("Óbitos") +
+  geom_point(aes(y = deaths), size = 0.4)
+ggplotly(graph.obitos, tooltip = "text")
+```
+
+  Curitiba se destaca atingindo a marca de 1388 óbitos, devendo ressaltar que o tamanho da população da cidade de Curitiba é superior aos demais municípios e, portanto o número de casos confirmados e consequentemente de óbitos será maior. Devido a isso podemos calcular até mesmo a taxa de mortalidade dessa doença, que tem como fórmula para esse cálculo $$\frac{\sum_{i=1}^{n}obitos}{\sum_{j=1}^{n} desfecho}$$ no qual a conotação "desfecho" é o número de recuperados + óbitos. Sendo assim tivemos uma baixa mortalidade para essa doença, no entanto o que preocupa as equipes da área da saúde principalmente é seu grande poder de proliferação, podendo deixar pessoas com grandes [sequelas no pós alta](http://saudedebate.com.br/noticias/sequelas-da-covid-19-a-vida-nao-voltou-ao-normal-apos-a-alta-hospitalar).
+
+## Óbitos para 100 mil habitantes com casos confirmados
+
+```{r echo=FALSE,message=FALSE,warning=FALSE}
+graph.obitos.p.100k <- df.cidades.analisadas %>% 
+  ggplot(mapping = aes(fill = city,x = date, y = deaths_per_100k_inhabitants, color = city,
+                       text = paste0(city, "\n",
+                                     date, "\n", 
+                                     "Óbitos p/ 100k habitantes: ", deaths_per_100k_inhabitants))) +
+  xlab("Dias/Meses") +
+  ylab("Óbitos por 100 mil habitantes") +
+  geom_point(aes(y = deaths_per_100k_inhabitants), size = 0.4)
+ggplotly(graph.obitos.p.100k, tooltip = "text")
+```  
+
+  Diferentemente dos comparativos de casos confirmados com a relação pelos 100 mil habitantes confirmados apresentados na segunda sessão desse material, não tivemos uma mudança abrupta nos posicionamentos das cidades, mas chama a atenção o distanciamento da capital do estado dos dois municípios de análise. 
+
```

Comment:
Acredito que o título comop "Óbitos por 100 mil ..." fique melhor.

Também me pareceu confuso se são óbitos por 100 mil habitantes ou 100 mil casos confirmados.
Se for casos confirmados, seria mais claro colocar no título
"Óbitos por 100 mil casos confirmados"